### PR TITLE
Add exercise rust-1-a

### PR DESCRIPTION
**Pull request by coriolinus**
This exercise is fairly straightforward. The follow-on exercise is planned
to extend this by passing in an arbitrary clock instead of a standard 12h
clock.

### CODE REVIEW

File: exercises/rust-1-a/example.rs [hunk 1]
```diff
@@ -0,0 +1,35 @@
+// input: the hour, 1-12 inclusive
+// output: seconds after the hour at which the hands are best aligned
+pub fn hands_match(mut hour: u8) -> u32 {
+    hour %= 12;
+    // let's solve this analytically.
+    //
+    // other plausible solution strategies:
```
Comment (ErikSchierboom) [inline exercises/rust-1-a/example.rs]: Do you have any idea how plausible these other approaches are?

Comment (coriolinus) [reply exercises/rust-1-a/example.rs]: I implemented the brute force version, which also took about 15 minutes. I didn't attempt binary search, but it seems plausible IMO.

Comment (ErikSchierboom) [reply exercises/rust-1-a/example.rs]: Good to hear!

**Comment by ErikSchierboom** [comment]
@iHiD Could you take a look too?

File: exercises/rust-1-a/README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+Your task is to find the moment when a clock's hands are perfectly aligned for a given hour. Given a standard 12-hour clock with hands which sweep continuously through time, there exists some second in each hour in which the hands are best lined up with each other. 
+
+We provide a method with the signature
+
+```rust
+fn hands_match(hour: u8) -> u32
+```
+
+The domain of `hour`s is `1-12` inclusive. The output must be the number of seconds after the hour at which the hands best line up.
```
Comment (iHiD) [inline exercises/rust-1-a/README.md]: While correct, "domain" is a bit too mathsy for my liking. I've reworded it for those without maths :)

```suggestion
The range of possible values for `hour` is 1-12. The output must be the number of seconds after the hour at which the hands best line up.
```

File: exercises/rust-1-a/README.md [hunk 3]
```diff
@@ -0,0 +1,22 @@
+## Introduction
+
+This is Part 1 of our research into how people write Rust in varying styles. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't look for a complete solution to the problem as this will negatively affect the research.
+
+## Instructions
+
+Your task is to find the moment when a clock's hands are perfectly aligned for a given hour. Given a standard 12-hour clock with hands which sweep continuously through time, there exists some second in each hour in which the hands are best lined up with each other. 
+
+We provide a method with the signature
```
Comment (iHiD) [inline exercises/rust-1-a/README.md]:
```suggestion
We provide a method with the signature:
```

**Comment by iHiD** [comment]
Great work :) I've added a couple of small comments.

**Comment by iHiD** [comment]
@coriolinus Could you follow up with an equivalent of https://github.com/exercism/research_experiment_1/pull/24 pls?

**Comment by coriolinus** [comment]
@iHiD https://github.com/exercism/research_experiment_1/pull/25

**Pull request merged**